.github/workflows/docker-promote.yml

```diff
@@ -58,24 +58,49 @@ jobs:
           cache-disabled: true
 
       - name: Docker upload
-        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_NAME }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" dockerUploadRelease
+        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_VERSION }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" dockerUploadRelease
 
       - name: Docker manifest
-        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_NAME }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" manifestDockerRelease
+        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_VERSION }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" manifestDockerRelease
 
   docker-verify:
-    needs: [validate, docker-promote]
+    needs: [validate,docker-promote]
     env:
-      RELEASE_NAME: ${{ needs.validate.outputs.release_name }}  # Use the output from the pre_process_release job
-    runs-on: ubuntu-22.04
-    permissions:
-      contents: read
-      actions: write
+      CONTAINER_NAME: besu-check
+      RELEASE_VERSION: ${{ needs.validate.outputs.release_version }}
+    runs-on: ${{ matrix.combination.runner }}
+    timeout-minutes: 4
+    strategy:
+      matrix:
+        combination:
+          - tag: ${{ needs.validate.outputs.release_version }}
+            platform: ''
+            runner: ubuntu-22.04
+          - tag: ${{ needs.validate.outputs.release_version }}-amd64
+            platform: 'linux/amd64'
+            runner: ubuntu-22.04
+          - tag: latest
+            platform: ''
+            runner: ubuntu-22.04
+          - tag: ${{ needs.validate.outputs.release_version }}-arm64
+            platform: ''
+            runner: besu-arm64
     steps:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```

Comment:
Not directly relevant to your changes but wondering if we really need to checkout the whole repo here? 
Is it just to get the script? I assume it pulls from our dockerhub so doesn't need the source to verify. Wonder if we can get hold of just the script instead.

Reply:
Parse checkout is possible. Wonder it is slower than the full checkout. Let me check this and switch to parse checkout only the file.

Reply:
Looks like parse checkout is faster. Let me update this.

Reply:
Updated